bindings/go/nvml/nvml.go

```diff
@@ -281,6 +281,24 @@ func GetDriverVersion() (string, error) {
 	return systemGetDriverVersion()
 }
 
+func GetCudaDriverVersion() (uint, uint, error) {
+	version, err := systemGetCudaDriverVersion()
+	if err != nil {
+		return 0, 0, err
+	}
+
+	// NVML_CUDA_DRIVER_VERSION_MAJOR ( v ) ((v)/1000)
+	// See https://docs.nvidia.com/deploy/nvml-api/group__nvmlSystemQueries.html#group__nvmlSystemQueries_1g40a4eb255d9766f6bc4c9402ce9102c2
+	versionLines := strings.Split(fmt.Sprintf("%g", float64(version) / 1000), ".")
+	if len(versionLines) < 2 {
+		return 0, 0, fmt.Errorf("GetCudaDriverVersion: Invalid version")
+	}
+
+	major, err := strconv.Atoi(versionLines[0])
+	minor, err := strconv.Atoi(versionLines[1])
+	return uint(major), uint(minor), err
+}
```

Comment:
All of this seems not really good but I don't know how it can better. The aim is to convert "10010" to 10 and 1 (for version 10.1). If anyone has an idea I'm interested.

I don't know if I should move all this logic in bindings.go instead ?

Reply:
https://github.com/NVIDIA/libnvidia-container/blob/master/src/driver.c#L397-L398

Reply:
Sounds really better, thank you